src/list.json

```diff
@@ -152,7 +152,7 @@
         "description": "Contribute to the mentioned Papoy projects and get a limited edition swag delivered to you hohoho!",
         "no_of_prs": 4,
         "link": "https://github.com/papoy-xor"
-      },
```

Comment:
please remove the  comma `,` this is causing the error

Reply:
already deleted

Reply:
Then push the new changes please.